test/speed-index.js

```diff
@@ -94,7 +94,6 @@ test('speed index calculate the right value', async t => {
 
 	const indexes = speedIndex.calculateSpeedIndexes(data.frames, data);
 	t.is(indexes.speedIndex, 1000);
-	t.is(indexes.perceptualSpeedIndex, 1000);
```

Comment:
this was somewhat nonsense since there we didn't actually have perceptual progress